R/subset_datarqst.R

```diff
@@ -15,9 +15,17 @@
 #' @param Species Character vector of six letter species codes. Include all species that are requested.
 #' @param filter Character vector of unique population species combinations (in the format PopID_Species).
```

Comment:
I guess I created these, but I think we should specify to the user that they're alternative arguments (i.e. you can either provide PopIDSpecies combos or provide species and pops separately)

Reply:
@LiamDBailey I have now updated the function description to clarify this to the user.